static/upnp/upnp-device-description.xml.in

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root xmlns="urn:schemas-upnp-org:device-1-0" configId="1">
+    <specVersion>
+        <major>1</major>
+        <minor>1</minor>
+    </specVersion>
+    <device>
+        <deviceType>urn:schemas-upnp-org:device:on-http:1</deviceType>
+        <friendlyName>RackHD</friendlyName>
+        <manufacturer>EMC Corporation</manufacturer>
+        <modelName>RackHD</modelName>
+        <modelURL>http://rackhd.readthedocs.org/en/latest/#</modelURL>

```

Comment:
At some point, I suspect it will - but they'll be redirecting for a while. They separated their admin interface at `.org` from the hosted content at `.io` and are redirecting now - apparently to resolve some XSS issues they were experiencing